src/i18n/strings/es.json

```diff
@@ -28,5 +28,7 @@
     "%(brand)s uses advanced browser features which aren't supported by your current browser.": "%(brand)s usa funciones avanzadas que su navegador actual no soporta.",
     "Powered by Matrix": "Funciona con Matrix",
     "Use %(brand)s on mobile": "Usar %(brand)s en modo móvil",
-    "Decentralised, encrypted chat &amp; collaboration powered by $matrixLogo": "Conversaciones y colaboración descentralizadas y cifradas gracias a $matrixLogo"
+    "Decentralised, encrypted chat &amp; collaboration powered by $matrixLogo": "Conversaciones y colaboración descentralizadas y cifradas gracias a $matrixLogo",
+    "Anyone can knock on this room to join. You can change this at any time from room settings.": "Anyone can knock on this room to join. You can change this at any time from room settings.",
+    "Knocking": "Knocking"
```

Comment:
We don't change the translation files manually - just use whatever `yarn i18n` gives you.

Reply:
Agh! I did actually ran `yarn i18n` in element-web and these translations were not included in react-sdk so I thought that ot should be included in the PR😅

Reply:
for this repo, your PR should be in labs.md :)